elements/pf-chip/pf-chip-group.ts

```diff
@@ -0,0 +1,264 @@
+  get hasCategory() {
+    return (this._categorySlotted || []).length > 0;
+  }
+
+  get remaining() {
+    return this.#chips.length - this.numChips;
+  }
+
+  /**
+   * updates chips when they change
+   */
+  #handleChipsChanged() {
+    const oldButtons = [...(this.#buttons || [])];
+    this.#chips = [...this.querySelectorAll('pf-chip:not([slot]):not([overflow-chip])')] as PfChip[];
+    const button = this._overflowChip?.button as HTMLElement;
+    const buttons = this.#chips.map(chip => chip.button as HTMLElement);
+    this.#buttons = [...buttons, button, this.button] as HTMLElement[];
+    this.#buttons = this.#buttons.filter(button => !!button);
+    if (oldButtons.length !== this.#buttons.length || !oldButtons.every((element, index) => element === this.#buttons[index])) {
+      if (this.#itemsInit) {
+        this.#tabindex.updateItems(this.#buttons);
+      } else {
+        this.#tabindex.initItems(this.#buttons);
+      }
+    }
+    this.#itemsInit = true;
+    this.#updateChips();
+  }
+
+  /**
+   * handles a new chip's `chip-ready` event
+   */
+  #onChipReady() {
+    this.#handleChipsChanged();
+  }
+
+  /**
+   * handles a chip's `chip-remove` event
+   */
+  async #onChipRemoved(event: Event) {
+    await this.updateComplete;
+    this.#handleChipsChanged();
+    this.focusOnChip(this.activeChip);
+  }
+
+  /**
+   * handles chip group close
+   */
+  #onCloseClick() {
+    /**
+     * @fires chip-group-remove
+     */
+    this.dispatchEvent(new Event('chip-group-remove', { bubbles: true }));
+    this.remove();
+  }
+
+  /**
+   * handles overflow chip's click event
+   */
+  async #onMoreClick(event: Event) {
+    event.stopPropagation();
+    this.open = !this.open;
+    await this.updateComplete;
+    this.#handleChipsChanged();
+    /**
+     * @fires overflow-chip-click
+     */
+    this.dispatchEvent(new Event('overflow-chip-click', event));
+  }
+
+  #onSlotchange() {
+    this.#updateHasCategory();
+  }
+
+  /**
+   * updates which chips are hidden
+   */
+  #updateChips() {
+    this.#chips.forEach((chip, i) => {
+      chip.closeLabel = this.closeLabel;
+      const overflowHidden = i >= this.numChips && !this.open;
+      if (overflowHidden) {
+        chip.setAttribute('overflow-hidden', 'overflow-hidden');
+      } else {
+        chip.removeAttribute('overflow-hidden');
+      }
+    });
+    this.requestUpdate();
+  }
+
+  /**
+   * updates whether chip group has a category
+   */
+  #updateHasCategory() {
+    if (this.hasCategory) {
+      this.setAttribute('has-category', 'has-category');
+    } else {
+      this.removeAttribute('has-category');
+    }
+  }
+
+  /**
+   * sets focus on active chip
+   */
+  focus() {
+    this.#tabindex.focusOnItem(this.#tabindex.activeItem);
+  }
+
+  /**
+   * makes chip active and sets focus on it
+   */
+  focusOnChip(chip: HTMLElement) {
+    const button = chip?.shadowRoot?.querySelector('button') as HTMLElement;
+    this.#tabindex.focusOnItem(button);
+  }
```

Comment:
Yes. 

One intentional overrides the element's focus(), so that focus is set on active element.  I often do this with complex elements so that we can control what happens when a consumer choses to focus on it.

The other sets focus to a particular chip.